File: codeforces/a-sum_trial1.js
try:
    x = int(input())
    if x<1 or x>9261:
        print("Invalid input")
        exit(1)
    for _ in range(x):
        a,b,c = [int(x) for x in input().split()]
        print( "YES" if 2*max(a,b,c) == a+b+c else "NO")
except:
    print("Invalid input")
    exit(1)

File: codeforces/a-sum_trial1.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int x;
    if (!(cin >> x) || x < 1 || x > 9261) {
        cout << "Invalid input" << endl;
        return 1;
    }
    for (int i = 0; i < x; ++i) {
        int a, b, c;
        if (!(cin >> a >> b >> c)) {
            cout << "Invalid input" << endl;
            return 1;
        }
        int max_val = max({a, b, c});
        int sum_val = a + b + c;
        if (2 * max_val == sum_val) {
            cout << "YES" << endl;
        } else {
            cout << "NO" << endl;
        }
    }

    return 0;
}